spec/controllers/home_controller_spec.rb

```diff
@@ -16,7 +16,7 @@
       end
 
       it "should default to html" do
-        expect(response.content_type).to eq("text/html")
+        expect(response.media_type).to eq('text/html')
```

Comment:
I question whether we still need these tests, knowing that (per Deprecation warning: 
`Rails 6.1 will return Content-Type header without modification.`). Regardless I updated them.

Reply:
It's probably worth checking how rspec sets up the get request, specifically the `content_type` header. If it's a sound default then we can probably skip this test, perhaps useful https://relishapp.com/rspec/rspec-rails/docs/request-specs/request-spec#requesting-a-json-response